entities/src/main/java/com/flipkart/varadhi/entities/VaradhiTopic.java

```diff
@@ -27,19 +29,22 @@ public class VaradhiTopic extends LifecycleEntity implements AbstractTopic {
      * @param capacity       the capacity policy of the topic
      * @param internalTopics the internal topics associated with this topic
      * @param status         the status of the topic
+     * @param nfrStrategy    the optional nfrStrategy
```

Comment:
not enough documentation for the param nfrStrategy.  When it is optional, what is the content of the param? 
it also does not tell what is the value of this param.